src/main/java/org/opensearch/ad/AnomalyDetectorProfileRunner.java

```diff
@@ -11,11 +11,11 @@
 
 package org.opensearch.ad;
 
-import static org.opensearch.ad.constant.CommonErrorMessages.FAIL_TO_FIND_DETECTOR_MSG;
-import static org.opensearch.ad.constant.CommonErrorMessages.FAIL_TO_PARSE_DETECTOR_MSG;
+import static org.opensearch.ad.constant.ADCommonMessages.FAIL_TO_PARSE_DETECTOR_MSG;
```

Comment:
What if there is an artifact which is not a detector? Say a clustering, or a density/hot-spot measurement? Maybe have "FAIL_TO_PARSE_ARTEFACT_MSG" ?

Reply:
I have already did similar things. But when I try to bring the change to current PR, I will need other dependency changes. To limit the scope of the PR, I used a temporary workaround like the above.